### PR TITLE
fixing builds

### LINKED ISSUE #25: correct the build and mark as broken!
This is incomplete. Please see the docs about `__osx` run requirement. You'll have to mark this build as broken after you have fixed it.

_Originally posted by @isuruf in https://github.com/conda-forge/libtvm-feedstock/pull/23#discussion_r825580493_

### PR DESCRIPTION
**Pull request by ngam**
fixes #25 

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by ngam** [comment]
@conda-forge-admin, please rerender

**Comment by ngam** [comment]
@conda-forge/core I am getting a "note" and a "warning" about needing a target of macos 11.0 or newer, but not an error. I am not really sure what's the best course of action here. Previously, I set the target for 11 and called it a day, but maybe that goes against the policies/procedures around here? So could someone please give this a look and help? Thanks!

cc @marcelotrevisani if you'd like to weigh in since you're part of core as well :) 

```
$SRC_DIR/src/runtime/contrib/bnns/bnns_json_runtime.cc:307:73: note: enclose 'BNNSDataLayoutVector' in a __builtin_available check to silence this warning
    auto dst_view = TView::as_is(dst_t).extract_outer_dim().with_layout(BNNSDataLayoutVector);
                                                                        ^~~~~~~~~~~~~~~~~~~~
$SRC_DIR/src/runtime/contrib/bnns/bnns_json_runtime.cc:313:53: warning: 'BNNSDataLayoutVector' is only available on macOS 11.0 or newer [-Wunguarded-availability-new]
      bias_view = TView::as_is(bias_md).with_layout(BNNSDataLayoutVector);
                                                    ^~~~~~~~~~~~~~~~~~~~
/Applications/Xcode_12.4.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.0.sdk/System/Library/Frameworks/Accelerate.framework/Frameworks/vecLib.framework/Headers/BNNS/bnns_constants.h:1134:3: note: 'BNNSDataLayoutVector' has been marked as being introduced in macOS 11.0 here, but the deployment target is macOS 10.9.0
  BNNSDataLayoutVector                         __API_AVAILABLE(macos(11.0), ios(14.0), watchos(7.0), tvos(14.0)) = 0x10000,
  ^

```

File: recipe/conda_build_config.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 # see this: https://github.com/phracker/MacOSX-SDKs/search?q=%3CMetal%2FMTLBlitCommandEncoder.h%3E
 # also some symbols only available after 11.0
 # Please consult conda-forge/core before doing this
-MACOSX_DEPLOYMENT_TARGET:  # [osx and x86_64]
-  - "11.0"                 # [osx and x86_64]
+# MACOSX_DEPLOYMENT_TARGET:  # [osx and x86_64]
+#   - "11.0"                 # [osx and x86_64]
```
Comment (isuruf) [inline recipe/conda_build_config.yaml]: You need to keep this uncommented

Comment (isuruf) [reply recipe/conda_build_config.yaml]: You also need to add `__osx` line as described in our docs to the run requirements.

Comment (ngam) [reply recipe/conda_build_config.yaml]: Okay, so do this regardless if it is an error or just a warning, right?

**Comment by ngam** [comment]
@conda-forge-admin, please rerender

File: recipe/conda_build_config.yaml (hunk 1 above)
Comment (ngam) [reply recipe/conda_build_config.yaml]: Ugh... maybe this is why I ended up not doing this to begin with... I never keep track:

```
The reported errors are:
- Encountered problems while solving:
-   - nothing provides requested __osx >=11.0
```

The azure pipeline has this osx info:

```
Starting: Initialize job
Agent name: 'Azure Pipelines 2'
Agent machine name: 'Mac-1648091671806'
Current agent version: '2.200.2'
Operating System
Mac OS X
10.15.7
19H1713

```

what do you advise here?

Comment (ngam) [reply recipe/conda_build_config.yaml]: Need to change this parameter:

https://github.com/ngam/libtvm-feedstock/blob/989c2a8b3eb223886d5118d6ee58b21bcf47ff64/.azure-pipelines/azure-pipelines-osx.yml#L8

i will try to improvise...

**Comment by ngam** [comment]
@conda-forge-admin, please rerender

File: conda-forge.yml [hunk 2]
```diff
@@ -7,3 +7,8 @@ test_on_native_only: true
 github:
   branch_name: main
   tooling_branch_name: main
+azure:
+  settings_osx:
+    pool:
+      vmImage: macOS-11
+
```
Comment (ngam) [inline conda-forge.yml:14]: we should document this

**Pull request merged**